src/styles/styles.scss

```diff
@@ -32,6 +32,7 @@ html {
     min-height: 100vh;
     padding-left: $BodyPaddingLeftRight;
     padding-right: $BodyPaddingLeftRight;
+    box-sizing: content-box;
```

Comment:
ouch, this is a global change.  Have you gone though the entire app to confirm this doesn't alter anything else?  Be sure to look at the busy/spinny background and how it fits the screen.  everything!

But I think you should make this change apply specifically to the case referenced in the ticket.